app/views/roadmap/index.html.erb

```diff
@@ -0,0 +1,116 @@
+<% content_for :title, t('roadmap.page_title') %>
+<% content_for :extra_headers do %>
+  <meta name="description" content="<%= t('roadmap.hero.description') %>" />
+<% end %>
+
+<div class="govuk-width-container">
+  <main id="content" class="govuk-main-wrapper">
+    <section class=" govuk-!-padding-bottom-6 govuk-!-margin-bottom-6 govuk-roadmap-section">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <%= render "govuk_publishing_components/components/title", title: t('roadmap.hero.heading'), margin_top: 0 %>
+          <p class="govuk-body"><%= t('roadmap.hero.description') %></p>
+        </div>
+
+        <div class="govuk-grid-column-one-third">
+          <%= render "image" %>
+        </div>
+      </div>
+    </section>
+    <section class="govuk-roadmap-section govuk-!-padding-bottom-4 govuk-!-margin-bottom-6">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <%= render "govuk_publishing_components/components/heading", {
+            text: t('roadmap.numbers_section.heading_large'),
+            margin_bottom: 0,
```

Comment:
Isn't the default for this component to have zero margin_bottom?

Reply:
(see also line 76)

Reply:
Yeeeep, you are correct, thanks for pointing that out. I'm not sure why I did that in the first place.
Have removed the override.